sections/04_study.md

```diff
@@ -74,7 +74,7 @@ computational restrictions.
 
 Epigenetic data, combined with deep learning, may have sufficient explanatory
 power to infer gene expression. For instance, a convolutional neural network
-applied to histone modifications, termed DeepChrome, [@tag:Singh2016_deepchrome]
+applied to histone modifications, termed DeepChrome [@tag:Singh2016_deepchrome]
```

Comment:
Changed instead to "For instance, the DeepChrome CNN [@tag:Singh2016_deepchrome] improved prediction accuracy of high or low gene expression from histone modifications over existing methods." in michaelmhoffman/deep-review@8f43dbfbcdae1c72956840c48f34b13ebb2f4c15.